microshift_troubleshooting/microshift-troubleshoot-updates.adoc

```diff
@@ -8,20 +8,15 @@ toc::[]
 
 To troubleshoot {microshift-short} updates, use the following guide.
 
-//[IMPORTANT]
-//====
-//You can only update {microshift-short} from one minor version to the next in sequence. For example, you must update 4.14 to 4.15.
-//====
-
 include::modules/microshift-updates-troubleshooting.adoc[leveloffset=+1]
 
 //additional resources for troubleshooting module
 [role="_additional-resources"]
 .Additional resources
 * xref:../microshift_install_get_ready/microshift-greenboot.adoc#microshift-greenboot-systemd-journal-data_microshift-greenboot[Enabling `systemd` journal service data persistency]
 * xref:../microshift_troubleshooting/microshift-version.adoc#microshift-version[Checking the MicroShift version]
-* xref:../microshift_install/microshift-install-rpm.adoc#microshift-service-stopping_microshift-install-rpm[Stopping the MicroShift service]
-* xref:../microshift_install/microshift-install-rpm.adoc#microshift-service-starting_microshift-install-rpm[Starting the MicroShift service]
+* xref:../microshift_install_rpm/microshift-install-rpm.adoc#microshift-service-stopping_microshift-install-rpm[Stopping the MicroShift service]
+* xref:../microshift_install_rpm/microshift-install-rpm.adoc#microshift-service-starting_microshift-install-rpm[Starting the MicroShift service]
 * link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/index[Composing, installing, and managing RHEL for Edge images]
 * link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/composing_installing_and_managing_rhel_for_edge_images/index#rolling_back_rhel_for_edge_images[Rolling back RHEL for Edge images]
 
```

Comment:
I know that `microshift-check-journal-logs-updates` isn't included in this PR, but in the preview, I noticed some strange formatting that you might want to double check. Notice the arrows in the callout text:
<img width="741" height="356" alt="Screenshot 2025-09-16 at 11 14 41 AM" src="https://github.com/user-attachments/assets/249c4278-d9c4-4106-b0e6-76aa53aa209b" />